ynput/ayon-launcher
# Webactions: Validate used user

## Changelog Description
AYON launcher validates that the user who triggered the action is the one logged in AYON launcher.

## Additional info
This should handle an edge case that 2 different users trigger webaction on the same machine. In that case it should ask for logic of the user who triggered the action.

## Testing notes:
- Step 1: Use different user from currently logged in user.
1. Prepare user1 and user2 on AYON server.
2. Login in AYON launcher to user1.
3. Login to AYON server as user2 and trigger webaction.
4. AYON launcher should ask to login, and will force to use the username.
- Step 2: Use 'Login with AYON server' button
5. In tray logout from AYON launcher.
6. In default browser log as user1, in second browser (e.g. Edge, Chrome) log as user2.
7. Start webaction from the second browser.
8. AYON login dialog should show up, use `Login with AYON server` button.
9. Confirm in default browser.
10. AYON launcher dialog should tell you that different user was used.
